File: c++/soma-impares.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;
int main() {
    int x, y, aux, soma, i;

    soma = 0;

    cout << "Digite dois numeros: " << endl;
    cin >> x;
    cin >> y;

    if (x > y)
    {
        aux = x;
        x = y;
        y = aux;
    }

    for (i = x+1; i < y; i++)
    {
        if (i % 2 != 0)
        {
            soma = soma + i;
        }
        
    }
    
    cout << "SOMA DOS IMPARES = " << soma << endl;

    return 0;
}

File: python/soma-impares.py
x: int
y: int
i: int
aux: int
soma: int

soma = 0

print("Digite dois numeros:")
x = int(input())
y = int(input())

if x > y:
    aux = x
    x = y
    y = aux

for i in range (x+1, y):
    if i % 2 != 0:
        soma = soma + i

print(f"SOMA DOS IMPARES = {soma}")